user-guide/Standard_Apps/Monitoring_Solutions/DataMiner_Objects_Tool/Using_DataMiner_Objects_Tool.md

```diff
@@ -0,0 +1,60 @@
+---
+uid: Using_DataMiner_Objects_Tool
+---
+
+# Using the DataMiner Objects Tool
+
+## Low-Code App
```

Comment:
Because so much of the content is blurred in the images under the Low-Code App title here, they don't look very nice and they don't give a very good first impression of what the app looks like. Could you instead show an image of the full app UI, and use numbers to indicate the different areas you want to refer to in the text (the way it's for example done here: https://docs.dataminer.services/user-guide/Standard_Apps/Monitoring_Solutions/Huawei_Manager/Working_with_Huawei_Manager_Dashboard.html)?